test/runnable/class_destructors.d

```diff
@@ -44,9 +43,11 @@ void testDeleteD()
 	char[] res;
 	ChildD cd = new ChildD();
 	cd.ptr = &res;
-	delete cd;
-
-	assert(res == "BA", cast(string) res);
+	if (!__ctfe)
```

Comment:
Why?

Reply:
Because the source to `rt_finalize` is not available at compile-time.